diesel/src/query_builder/upsert/on_conflict_actions.rs

```diff
@@ -51,10 +72,11 @@ where
     }
 }
 
-impl<DB, T, SP> QueryFragment<DB, SP> for DoUpdate<T>
+impl<DB, T>
+    QueryFragment<DB, crate::backend::sql_dialect::on_conflict_clause::PgLikeOnConflictClause>
```

Comment:
Again this breaks the sqlite implementation. (Fixing this can be done by restricting `SP` to the same trait as above)

Reply:
So sorry but I think I'm misunderstanding this. By breaking do you mean failing to compile? Because I at least see that 
```
cargo test --no-default-features --features "extras sqlite"
```

finishes successfully for me 😅